nightly/expensive.txt

```diff
@@ -56,65 +56,66 @@ expensive --timeout=500 near-client near_client tests::consensus::test_consensus
 expensive --timeout=500 near-client near_client tests::consensus::test_consensus_with_epoch_switches --features nightly_protocol,nightly_protocol_features
 
 # testnet rpc
-expensive nearcore test_tps_regression test::test_highload
-expensive nearcore test_tps_regression test::test_highload --features nightly_protocol,nightly_protocol_features
+# Is disabled, because it has `#[ignore]`
+# expensive nearcore test_tps_regression test::test_highload
+# expensive nearcore test_tps_regression test::test_highload --features nightly_protocol,nightly_protocol_features
```

Comment:
For the purposes of NayDuck ignored tests are passing so this should stay enabled. Alternatively, disable this test in NayDuck but then remove `#[ignore]`. Best to disable tests in single location. Same for `test_seal_removal`.

Reply:
@mina86 Does nayduck run all tests with `#[ignore]`?

Reply:
NayDuck runs what it’s configured to run and yes, all `#[ignore]` tests are currently listed.

Reply:
@mina86 Ok, I undid this change. I'll assume it's ok to have here tests with `#[ignore]`.